src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task{
```

Comment:
Great Job at coding standards over here! Although maybe consider moving the public final constants to main and making them private.